src/App.js

```diff
@@ -1,12 +1,75 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
 import './App.css';
+import './components/InfoCard.css';
+import pokemonLogo from "./assets/1200px-International_Pokémon_logo.svg.png";
+import InfoCard from "./components/InfoCard";
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    // Beginpunt initialiseren voor range van 20 op te vragen pokemon-objecten:
+    const [rangeStart, setRangeStart] = useState(0);
+
+    // Leeg object maken voor 20 pokemon-objecten:
+    const [pokeNamesData, setPokeNamesData] = useState({});
+
+
+    useEffect(() => {
+
+        console.log("rangeStart is geïnitialiseerd of aangepast");
+        console.log(`de range pokemon-objecten is nu: ${rangeStart} t/m ${rangeStart + 20}`);
+
+        // Get request met als resultaat een object met alle 20 gevonden pokemon-objecten
+        // met daarin onder andere een .name-key waarin de naam als string staat.
+        // Gebruik maken van de waarde van rangeStart-state.
+        async function fetchDataNames() {
+            try {
+
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon?limit=20&offset=${rangeStart})`);
+                console.log(result);
+
+                setPokeNamesData(result.data.results);
+
+                console.log("fetchDataNames is uitgevoerd");
+                console.log("pokeNamesData bevat nu:" + pokeNamesData);
```

Comment:
Op het moment dat setPokeNamesData geinvoked wordt, dan moet er eerst een rerender overheen wil pokeNamesData kunnen uitlezen. Hier heeft iemand precies dezelfde vraag:
https://stackoverflow.com/questions/54069253/usestate-set-method-not-reflecting-change-immediately